File: CS/WindowsFormsApplication5/CalcEditDescendant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Popup;
using System.ComponentModel;

namespace WindowsFormsApplication5
{
    [ToolboxItem(true)]
    public class CalcEditDescendant : CalcEdit
    {
        public CalcEditDescendant()
        {

        }
        protected override DevExpress.XtraEditors.Popup.PopupBaseForm CreatePopupForm()
        {
            return new PopupCalcEditFormDescendant(this); //base.CreatePopupForm();
        }
    }
}

File: VB/WindowsFormsApplication5/CalcEditDescendant.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Popup
Imports System.ComponentModel

Namespace WindowsFormsApplication5
	<ToolboxItem(True)> _
	Public Class CalcEditDescendant
		Inherits CalcEdit
		Public Sub New()

		End Sub
		Protected Overrides Function CreatePopupForm() As DevExpress.XtraEditors.Popup.PopupBaseForm
			Return New PopupCalcEditFormDescendant(Me) 'base.CreatePopupForm();
		End Function
	End Class
End Namespace
